### PR TITLE
Week 3 - Karin Nordkvist (Secrets Anonymous site)

### PR DESCRIPTION
**Pull request by karinnordkvist**
A responsive, dark-mode site on secrets - featuring a form, accordion FAQ and a video for hero.

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,7 +1,315 @@
...
+.accordion-item-header::after {
+  content: "\25B8";
+  position: absolute;
+  right: 0px;
+  transition: transform .3s ease-in-out;
+}
+
+.accordion-item-header.active::after {
+  transform: rotate(90deg);
+}
```
Comment (Cilpersson) [inline code/style.css:249]: Nice to see that you didn't give up even though ::after felt a bit confusing for you at first 💪

File: code/style.css [hunk 2]
```diff
@@ -1,7 +1,315 @@
...
+button {
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  display: block;
+  margin: 40px auto 20px auto;
+  border: 1px solid #fff;
+  color: #fff;
+  background: none;
+  outline: none;
```
Comment (Cilpersson) [inline code/style.css:199]: I know it's sooo tempting to remove the outline (they're not pretty), but don't. It makes your page less web accessible, when you tab through your site you can't see when the button is highlighted 😭 This is important for people using screen readers or cant use a mouse when browsing your page. I believe you will talk more about web accessibility in a few weeks 👍

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,315 @@
...
+  text-transform: uppercase;
+}
+
+* {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
```
Comment (Cilpersson) [inline code/style.css:59]: Keep the `* {...}`-styling at the top of the stylesheet ✌️

**Review by Cilpersson — APPROVED** [review]
Karin you have some clean looking code. KEEP IT UP! The structure of your code is A-class and it's so nice to see. You have good naming for your classes and I enjoyed reading your readme, it was short and to the point but not lacking reflections. Well-written and clean code is something companies value highly!  

I have left comments for you, hopefully you will take my reflections with you for the upcoming projects. Your commits are good, but I would love to see more. Try to commit at least 3-4 times per coding session, when it comes to commits **less is a bore** 🥳

Great job, BE PROUD and don't forget to CELEBRATE!

File: code/index.html [hunk 4]
```diff
@@ -3,28 +3,167 @@
...
+        
+        <label for="secret">What is your secret?*</label>
+        <textarea 
+          id="secret" 
+          name="secret" 
+          minlength="10" 
+          maxlength="500" 
+          placeholder="Spill the beans here..."
+          required
+          ></textarea>
```
[the remark this answers is not in the document]
Comment (karinnordkvist) [reply code/index.html:57]: I have to give all credit to @EmOlsz for this one 😅

**Comment by karinnordkvist** [comment]
Thank you so much @Cilpersson for your comments and thoughts. All that praise gives me goosebumps and your words definitely made my day. I will take all improvement suggestions into my future work, for sure. Thanks, again! 😍

**Pull request closed without merging**